docs/manual/java/guide/production/ProductionOverview.md

```diff
@@ -1,24 +1,56 @@
+    * Otherwise, you can configure `sbt-native-packager` to produce [other archive formats](https://www.scala-sbt.org/sbt-native-packager/formats/universal.html#build), [Windows (MSI) installers](https://www.scala-sbt.org/sbt-native-packager/formats/windows.html), Linux packages in [Debian](https://www.scala-sbt.org/sbt-native-packager/formats/debian.html) or [RPM](https://www.scala-sbt.org/sbt-native-packager/formats/rpm.html) format and more.
 
-The following example shows how to set `contact-points` in the `application.conf` file:
+* Maven has a variety of plugins to produce artifacts for various platforms:
+
+    * The [Maven Assembly Plugin](http://maven.apache.org/plugins/maven-assembly-plugin/) can be used to build archives in zip, tar and other similar formats.
+
+    * The [Docker Maven Plugin](https://dmp.fabric8.io/) is recommended for building Docker images. Maven support in Lightbend Orchestration is currently under development.
+
+    * Other Maven packaging plugins in varying states of maturity are available from the [MojoHaus Project](https://www.mojohaus.org/plugins.html).
+
+* At runtime, services need to locate the addresses of other services to communicate with them. This requires you to configure an implementation of a [`ServiceLocator`](api/index.html?com/lightbend/lagom/javadsl/api/ServiceLocator.html) that Lagom uses to look up the addresses of services by their names. The production environment you choose might provide a service discovery mechanism that you can use with Lagom.
+
+    * For simple deployments, Lagom includes a built-in service locator that uses addresses specified in the service configuration ([described below](#Using-static-values-for-services-and-Cassandra)).
+
+    * Lightbend Orchestration provides an open-source [`ServiceLocator` implementation](https://developer.lightbend.com/docs/lightbend-orchestration/current/features.html#service-location) that integrates with the service discovery features of Kubernetes or DC/OS, or any other environment that supports service discovery via DNS.
+
+    * Otherwise, you can implement the interface yourself to integrate with a service registry of your choosing (such as [Consul](https://www.consul.io/), [ZooKeeper](https://zookeeper.apache.org/), or [etcd](https://coreos.com/etcd/)) or start with an open-source example implementation such as [`lagom-service-locator-consul`](https://github.com/jboner/lagom-service-locator-consul) or [`lagom-service-locator-zookeeper`](https://github.com/jboner/lagom-service-locator-zookeeper).
+
+* Services that require an [Akka Cluster](https://doc.akka.io/docs/akka/current/cluster-usage.html) (which includes any that use the Lagom [[Persistence|PersistentEntity]] or [[Publish-Subscribe|PubSub]] APIs) must have a strategy for forming a cluster or joining an existing cluster on startup.
```

Comment:
Maybe we are giving here the impression that once you use the Persistence API or the PubSub API you have to form a cluster and think about the implications of it. 

One can still use them my having a single node (actually, a cluster of one) and not caring about cluster formation. 

On the other hand, I think that people deploying single node clusters in orchestrate environments should be aware of this, because soon or later they will perform a rolling update and create a split without even realizing it.

Reply:
Even with a single node you still need to form the cluster of one, somehow, so you still need to care about this.

We don't really encourage deploying single instances of a service, but I think when we expand the cluster documentation in a future change, we can discuss this case specifically. In the past, we have seen people deploy several nodes of a service as single node clusters because it's easier. 😧Sometimes the problems aren't apparent immediately, so I think it's important to be clear about it.